install-deps.sh

```diff
@@ -262,6 +263,7 @@ if [ x$(uname)x = xFreeBSDx ]; then
         misc/e2fsprogs-libuuid \
         misc/getopt \
         net/socat \
+        net/rabbitmq-c \
```

Comment:
as `WITH_RADOSGW_AMQP_ENDPOINT=OFF` in do_freebsd.sh, we don't need this ?

Reply:
@tchaikov 
Like the topic says: matching to what is used in the FreeBSD ports file.
In do_freebsd.sh I do/did not test this, but perhaps I should as well.
But it does compile and as such it is available in the FreeBSD port.

Reply:
@wjwithagen i understand that you are trying to match the FreeBSD port file. but what i don't understand is the reason why you would want do so even if install-deps.sh is not adding something that do_freebsd.sh does not need. if what you want is but mirror port file, what would we need to keep the FreeBSD branch in this file? it's but adding our load of sync'ing with the port.

Reply:
@tchaikov 
I have 2 testing environments, and the ports environment:
 -  CLI for development
 - Jenkins for integrated testing
    To see if merged PRs are nog causing unexpected trouble
 - Ports is even more complex, because of the rules being set by the ports-admins.

And I keep running into stuff not installed because it is not in `install-deps.sh`.
So `install-deps.sh` should IMHO contain everything FreeBSD can provide to build as much of Ceph as possible. So that if flags are changed, you do not bomb out because a library is not there. Especially for those that are not all to familiar with the full details of Ceph it is a large dissatisfier, and puts people off.